bindings/pyroot/pythonizations/test/rtensor.py

```diff
@@ -11,40 +12,54 @@ def check_shape(root_obj, np_obj):
     return root_shape == np_shape
 
 
-
 class AsRTensor(unittest.TestCase):
     """
     Test AsRTensor adoption mechanism
     """
 
     # Helpers
-    dtypes = [
-        "int32", "int64", "uint32", "uint64", "float32", "float64"
-    ]
-    cpptypes = ["int", "long", "unsigned int", "unsigned long", "float", "double"]
+    cpptypes = {
+        "int32": "int32_t",
+        "int64": "int64_t" if platform.system() != "Darwin" else "long",
```

Comment:
I don't understand why the obvious mapping of `int64` to `int64_t` doesn't work on macOS, but on Windows and Linux it works fine.

Therefore, I'm treating macOS/Darwin as the special case now.